vtm-ios/build.gradle

```diff
@@ -45,6 +45,11 @@ task copyVtmThemesResources(type: Copy) {
     into("${buildDir}")
 }
 
+task copyVtmPlaygroundResources(type: Copy) {
+    from("../vtm-playground/resources")
+    into("${buildDir}/assets/")
+}
+
 
```

Comment:
Do we want that?
`vtm-ios` is a published library, while `vtm-playground` is a sample app.